Packs/FullHunt/Integrations/FullHunt/FullHunt.py

```diff
@@ -0,0 +1,252 @@
+            outputs_prefix='FullHunt.Host',
+            outputs_key_field='host',
+            outputs=host_data
+        ))
+
+    return command_results
+
+
+def get_domain_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    domain = args.get('domain')
+    if not domain:
+        raise ValueError('domain not specified')
+    elif ',' in domain:
+        raise ValueError('Several domains provided, please provide one unique domain')
+
+    domain_data = client.get_domain_details(domain)
+
+    readable_output = tableToMarkdown('Domain information', domain_data)
+
+    return CommandResults(
+        outputs_prefix='FullHunt.Domain',
+        outputs_key_field='',
+        outputs=domain_data,
+        readable_output=readable_output
+    )
+
+
+def get_subdomain_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    domain = args.get('domain')
+    if not domain:
+        raise ValueError('domain not specified')
+    elif ',' in domain:
+        raise ValueError('Several domains provided, please provide one unique domain')
+
+    subdomain_data = client.get_subdomain(domain)
+
+    readable_output = tableToMarkdown('Subdomains information', subdomain_data)
+
+    return CommandResults(
+        outputs_prefix='FullHunt.Subdomain',
+        outputs_key_field='',
+        outputs=subdomain_data,
+        readable_output=readable_output
+    )
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """
+    main function, parses params and runs command functions
+    """
+
+    params = demisto.params()
+    args = demisto.args()
+    command = demisto.command()
+
+    api_key = params.get('credentials', {}).get('password')
+
+    # Get the service API url
+    base_url = urljoin(params.get('url'), '/api/v1')
+
+    # if your Client class inherits from BaseClient, SSL verification is
+    # handled out of the box by it, just pass ``verify_certificate`` to
+    # the Client constructor
+    verify_certificate = not params.get('insecure', False)
+
+    # if your Client class inherits from BaseClient, system proxy is handled
+    # out of the box by it, just pass ``proxy`` to the Client constructor
+    proxy = params.get('proxy', False)
+
+    # Integration that implements reputation commands (e.g. url, ip, domain,..., etc) must have
+    # a reliability score of the source providing the intelligence data.
+    reliability = params.get('integrationReliability', DBotScoreReliability.C)
+
+    demisto.debug(f'Command being called is {command}')
+    try:
+        headers = {
+            'X-API-KEY': f'{api_key}'
+        }
+        client = Client(
+            base_url=base_url,
+            verify=verify_certificate,
+            headers=headers,
+            proxy=handle_proxy("proxy", proxy).get("http", ""))
+
+        if command == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            result = test_module(client, params)
```

Comment:
test_module returns a string. other commands return CommandResult.
Please assigne a different variable for test_module return value.
example:
test_module_result = test_module(...)

Reply:
Done and pushed